assets/js/modules/reader-revenue-manager/utils/validation.test.js

```diff
@@ -19,11 +19,8 @@
 /**
  * Internal dependencies
  */
-import {
-	isValidPublicationID,
-	isValidSnippetMode,
-	isURLUsingHTTPS,
-} from './validation';
+import { isValidPublicationID, isValidSnippetMode } from './validation';
+import { isURLUsingHTTPS } from '../../../util/is-url-using-https';
```

Comment:
@ankitrox, actually, we don't need to test this anymore as it doesn't belong to the validation file.

https://github.com/google/site-kit-wp/blob/cc37e4ec903eb091d2f6c3f298bffd43cf1ccbdd/assets/js/modules/reader-revenue-manager/utils/validation.test.js#L47-L72